features/support/paths.rb

```diff
@@ -262,6 +262,12 @@ def path_to(page_name)
     when /^the [mM]y account page$/
       '/my/account'
 
+    when /^the [aA]ccess [tT]oken page$/
+      '/my/access_token'
+
+    when /^the [sS]ettings page$/
```

Comment:
"settings page" is not a good name to refer to this one, because it can be confused with the settings page in the administration (`/settings`).

You are actually overriding an existing case this way: `when /^the(?: (.+?) tab of the)? settings page$/`